_includes/partners/contact.html

```diff
@@ -0,0 +1,35 @@
+<main id="main-content">
+  <section>
+    <div class="container partners-container">
+      <div class="grid-row">
+        <div class="tablet:grid-col-9 accordion-container">
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/shell.js"></script>
```

Comment:
The `//` means it might load over plain HTTP... which in production we'd never want, I'd prefer`https://` 

```suggestion
          <script charset="utf-8" type="text/javascript" src="https://js.hsforms.net/forms/shell.js"></script>
```